data/ecosystems/p/polkadot.toml

```diff
@@ -3912,6 +3917,12 @@ url = "https://github.com/polkadot-js/wasm"
 [[repo]]
 url = "https://github.com/polkadotters/SubAuction"
 
+[[repo]]
+url = "https://github.com/PolkaEX"
```

Comment:
should have its own ecosystem, can you remove this (since it is not a repo) and create a new toml file for itself and place "PolkaEX" under the `sub_ecosystems` at the top of this file?

Reply:
manually added to the database